src/metabase/api/activity.clj

```diff
@@ -112,11 +116,39 @@
 (def ^:private views-limit 8)
 (def ^:private card-runs-limit 8)
 
-(api/defendpoint GET "/recent_views"
+(api/defendpoint ^:deprecated GET "/recent_views"
   "Get a list of 100 models (cards, models, tables, dashboards, and collections) that the current user has been viewing most
   recently. Return a maximum of 20 model of each, if they've looked at at least 20."
   []
-  {:recent_views (recent-views/get-list *current-user-id*)})
+  (u/snake-keys
+   (recent-views/get-recents *current-user-id* :views)))
+
+(api/defendpoint GET "/recents"
+  "Get a list of recent items the current user has been viewing most recently under the `:recents` key.
+  Allows for filtering by context, either views or selections"
+  [:as {{:keys [context]} :params}]
+  ;; multiple qps of context
+  {context :string}
+  (let [context (if (str/blank? context)
+                  (throw (ex-info "Context required" {:context context}))
+                  (mapv keyword (str/split context #",")))]
```

Comment:
To pass a list of values for a query parameter, we typically repeat the param name like `GET api/recents?context=view&context=collection`.

See this for an example, with the `filtering` parameter:
https://github.com/metabase/metabase/blob/5b01cf86d7bd4730140fdf8e6c438690de697be6/src/metabase/api/dashboard.clj#L1086-L1115

Reply:
Nice helper!